test/test_all.py

```diff
@@ -145,27 +160,66 @@ def check_logs(configuration, golden_logs_parent_path, logs_subpath):
     assert logs == golden_logs
 
 
+def check_db(configuration):
+    """Run tests to determine whether the generated library database is as expected.
+
+    Keyword arguments:
+    configuration -- dictionary defining the libraries-repository-engine configuration
+    """
+    checksum_placeholder = "CHECKSUM_PLACEHOLDER"
+
+    # Load generated db
+    with pathlib.Path(configuration["LibrariesDB"]).open(mode="r", encoding="utf-8") as db_file:
+        db = json.load(fp=db_file)
+    for release in db["Releases"]:
+        # The checksum values in the db will be different on every run, so it's necessary to replace them with a
+        # placeholder before comparing to the golden master
+        release["Checksum"] = checksum_placeholder
+
+    # Load golden index
+    golden_db_template = test_data_path.joinpath("test_all", "golden", "db.json").read_text(encoding="utf-8")
+    # Fill in mutable content
+    golden_db_string = string.Template(template=golden_db_template).substitute(
+        base_download_url=configuration["BaseDownloadUrl"],
+        checksum_placeholder=checksum_placeholder,
+        git_clones_folder=configuration["GitClonesFolder"],
+    )
+    golden_db = json.loads(golden_db_string)
+
+    # Compare db against golden master
+    # Order of entries in the db is arbitrary so a simply equality assertion is not possible
+    assert len(db["Libraries"]) == len(golden_db["Libraries"])
+    for library in db["Libraries"]:
+        assert library in golden_db["Libraries"]
+
+    assert len(db["Releases"]) == len(golden_db["Releases"])
+    for release in db["Releases"]:
+        assert library in golden_db["Releases"]
```

Comment:
I guess this should be:
```suggestion
        assert release in golden_db["Releases"]
```
?

Reply:
Thanks! I have fixed it here:
https://github.com/arduino/libraries-repository-engine/compare/e29e9c2695d132e9e64083de6a0389818d001025..7ac77910a83f97e2e7aba20caa276978010b3eae